indra/llprimitive/llprimitive.h

```diff
@@ -498,12 +512,15 @@ class LLPrimitive : public LLXform
 
     void copyTEs(const LLPrimitive *primitive);
     S32 packTEField(U8 *cur_ptr, U8 *data_ptr, U8 data_size, U8 last_face_index, EMsgVariableType type) const;
+
+    S32  packTEMessageBuffer(U8* packed_buffer) const;
     bool packTEMessage(LLMessageSystem *mesgsys) const;
-    bool packTEMessage(LLDataPacker &dp) const;
```

Comment:
The `packTEMessage()` that expected an `LLDataPacker&` argument has been removed: it was never called anywhere.